src/datetimepicker.ios.js

```diff
@@ -9,12 +9,14 @@
  * @format
  * @flow strict-local
  */
+import React, {useEffect} from 'react';
 import {StyleSheet} from 'react-native';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
```

Comment:
I would rather not rely on these as I do not consider these to be part of the stable api